lib/version_utils.pm

```diff
@@ -89,44 +96,72 @@ our %EXPORT_TAGS = (
     SCENARIO => (SCENARIO)
 );
 
+#TODO can we remove this?
```

Comment:
Here I am not sure if we need this, we have a function definition later

Reply:
I would say: remove it.
That looks like someone merged a bad PR or some black perl I ignore.